chat/rich-links/src/RichLink/RichLink.spec.tsx

```diff
@@ -66,5 +68,32 @@ describe('@lg-chat/rich-links', () => {
       expect(screen.getByText('Link to Video')).toBeInTheDocument();
       expect(screen.getByText('Link to Website')).toBeInTheDocument();
     });
+
+    it('ignores unknown variants', () => {
```

Comment:
This test name is a little unclear. What does ignoring the unknown variant imply? Maybe you can add that it still renders?